src/main/java/com/linkedin/kmf/apps/configs/MirrorPipelineMonitorConfig.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.apps.configs;
+
+public class MirrorPipelineMonitorConfig {
+
+  public static final String CONSUME_SERVICE_CONFIG = "consume.props";
```

Comment:
Can you provide the specification of all these configs, i.e. required from user or not, default value if not required from user, as we did in TopicManagementServiceConfig? The class probably needs to extend `AbstractConfig` so that it can provide methods to read values. Then the methods such as `getDouble` which you defined in the `MirrorPipelineMonitor` won't be necessary.

BTW, in case it is helpful, in the future you may want to take a look at the existing code that does what you intend to do. For example, we already have method that reads partitionToBrokerRatio from user-provided config. And that is done without defining our own getDouble. Then you can decide what is the best choice for your code.